gnovm/stdlibs/std/emit_event.go

```diff
@@ -46,12 +59,20 @@ func attrKeysAndValues(attrs []string) ([]gnoEventAttribute, error) {
 }
 
 type gnoEvent struct {
-	Type       string              `json:"type"`
+	OrigCaller string              `json:"orig_caller"`
+	Prev       prevStack           `json:"prev"`
```

Comment:
Just call this PrevRealm

Reply:
https://github.com/gnolang/gno/blob/252577668f81e62062e24f63f2feca98ea60dccc/gnovm/stdlibs/std/emit_event.go#L24

Reply:
This should be called PrevRealm, and its type be of type `Realm` (to be created), reflecting Gno's internal Realm type.

```go
type Realm struct {
	Addr    crypto.Bech32Address `json:"addr"`
	PkgPath string `json:"pkg_path"`
}
```

This creates consistency with Gno's Realm type, and removes the redundant "Prev" in prevStack.

Reply:
8065db7ca0b7585e1b9b0fd507bdf5c5a5f32ae7